src/storybook/stories/docs/database-schema.md

```diff
@@ -0,0 +1,90 @@
+
+- `priority: int (maybe enum)` - The priority of the message, aka. INFO, WARNING, DANGER.
+
+  This should be a predefined list of values.
+
+- `title_key: varchar(128) | null` - The translation key for the title of the notification.
+
+- `message_key: varchar(128)` - The translation key for the notification message content
+
+- `meta: JSON` - data that doesn’t have to be queried, like icon or image information.
+
+  The `meta` field could be stored in another table, similar to other WordPress schemas. Another possibly is to use a concept of message type. A message could be of a type that has metadata attached to it, like icon.
+
+## Channel schema
+
+Core should have a set registered channels.
+
+Plugins can register channels of their own.
+
+### wp_notifications_channels table
+
+- `id: int` - The ID of the channel
+- `created_at: timestamp` - The timestamp of when the channel was created.
+- `updated_at: timestamp` - The timestamp of when the channel was last updated.
+- `source: varchar(128)` - The source of the channel, aka. the name of the plugin.
+- `name_key: varchar(128)` - The translation key for the name of the channel.
+- `description_key: varchar(128)` - The translation key for the description of the channel.
+- `role: varchar(128)` The minimum required role to subscribe to messages from the channel.
+- `meta: JSON` - Data that doesn’t have to be queried, like icon or images
+
+  Another way to add metadata to a channel could be the registration process in PHP and JS. Attaching icons could be done in code when calling something like `wp_notifications_register_channel`.
+
+## Message Queue
+
+Join table used to map messages to specific users. Could periodically be cleared depending on message `expires_at` and/or how long ago it was dismissed.
+
+The full history of messages for every user is retained and could be easily looked up.
+
+If a message has been orphan it can safely be deleted.
+
+### wp_notifications_queue table
+
+- `message_id: int` - The ID of the enqueued message.
+- `user_id: int` - The ID of the user.
+- `dismissed_at: timestamp | null` - The timestamp of when the notification was dismissed.
+- `displayed_at: timestamp | null` - The timestamp of when the notification was first displayed.
+
+  There could definitely be addition statuses, and if the time of the state update is deemed unnecessary, perhaps they could be a single enum rather that timestamps.
+
+## Subscriptions
+
+Join table used determine for whom to enqueue messages for a specific channel.
+
+Logic to authorize a users to subscribe to a channel would be based on a comparison of the `role` property of the channel and the user.
+
+### wp_notifications_subscriptions
+
+- `user_id: int` - The ID of the user subscribed to the channel.
+- `channel_id: int` - The ID of the channel subscribed to.
+- `snoozed_until: timestamp | null` - The optional timestamp of when to resume the channel.
```

Comment:
@erikyo this hasn't been merged into `develop` yet, you should probably add the original schema doc back.

Reply:
yep i know (seems the only thing that isn't merged correctly at the moment), this is due the pr was created time ago and wasn't synced properly!